floodsub.go

```diff
@@ -252,8 +252,8 @@ func (p *PubSub) announce(topic string, sub bool) {
 	}
 }
 
-// notifySubs sends a given message to all corresponding subscribbers.
-// Only called from processLoop.
+// notifySubs sends a given message to all corresponding subscribers.
+// Only called from maybePublishMessage.
```

Comment:
Keep this comment as is please too, the intention is to convey that it's in the event loop.
Btw, thanks for fixing the typo!

Reply:
Ah, ok, makes sense - done in 4643b97